server/rest/src/main/java/org/infinispan/rest/resources/ServerResource.java

```diff
@@ -466,6 +473,30 @@ private CompletionStage<RestResponse> dataSourceTest(RestRequest restRequest) {
       }, invocationHelper.getExecutor());
    }
 
+   private CompletionStage<RestResponse> getCacheConfigDefaultAttributes(RestRequest request) {
+      Configuration configuration = new ConfigurationBuilder().build();
+      Map<String, String> attributes = new LinkedHashMap<>();
+      allAttributes(configuration, attributes);
+      return asJsonResponseFuture(Json.make(attributes), isPretty(request));
+   }
+
+   private static void allAttributes(ConfigurationElement<?> element, Map<String, String> attributes) {
+      for (Attribute<?> attribute : element.attributes().attributes()) {
+            AttributeDefinition<?> definition = attribute.getAttributeDefinition();
+            String value;
+            try{
+               value = attribute.getInitialValue().toString();
+            } catch (Exception e){
```

Comment:
We should just be able to check if the initial value is null or not and not catch all Exception.